pwndbg/dbg/gdb.py

```diff
@@ -192,7 +218,10 @@ def fetch_lazy(self) -> None:
 
     @override
     def __int__(self) -> int:
-        return int(self.inner)
+        try:
+            return int(self.inner)
+        except gdb.error as e:
+            raise pwndbg.dbg_mod.Error(e)
```

Comment:
When can this fail? Why was it that we did not handle this exception in the past? Same for others.